.circleci/config.yml

```diff
@@ -28,9 +28,9 @@ store-log-artifacts: &store-log-artifacts
     path: log
     destination: log
 
-npm-cache-key: &npm-cache-key node-v10-{{ checksum "yarn.lock" }}-5
-bundle-cache-key: &bundle-cache-key v2-bundler-gems-{{ .Environment.DB }}-{{ arch }}-{{ .Branch }}-{{ checksum "Gemfile.lock" }}
-assets-cache-key: &assets-cache-key v1-asset-cache-{{ checksum "tmp/assets_related_checksums" }}
+npm-cache-key: &npm-cache-key node-v10-{{ checksum "yarn.lock" }}-{{ .Environment.CACHE_VERSION }}
+bundle-cache-key: &bundle-cache-key bundler-gems-{{ .Environment.DB }}-{{ arch }}-{{ .Branch }}-{{ checksum "Gemfile.lock" }}-{{ .Environment.CACHE_VERSION }}
```

Comment:
I found a bug here: `{{ .Environment.DB }}` is empty and it's rendered as `<no value>`. So the intention was to have different bundles cached for different DBs but they all share the same bundle cache.

Reply:
Why not remove `{{ .Environment.DB }}` then?

Also how do we use this `.Environment.CACHE_VERSION`?

Reply:
ok, got it, but version should be at the beginning, othervice it will not be effective because of partial matching I think. So maybe replace these `v1`, `v2` prefixes.

Reply:
> Why not remove `{{ .Environment.DB }}` then?
> 

I think we should do that in another PR

Reply:
Read circleci docs about caching, IIRC if key matches the beginning of another key, then it is considered a match. But don't remember a link and have no time to search now. If you figure out things are ok and you get a review from somebody else, that's fine with me. As I will be available next week.